--- /usr/local/lib/python3.6/site-packages/youtube_dl/extractor/youtube.py.orig 2017-04-28 03:43:29.956014198 +0000
+++ /usr/local/lib/python3.6/site-packages/youtube_dl/extractor/youtube.py      2017-04-28 03:45:32.816497248 +0000
@@ -963,7 +963,7 @@
 
     def _extract_signature_function(self, video_id, player_url, example_sig):
         id_m = re.match(
-            r'.*?-(?P<id>[a-zA-Z0-9_-]+)(?:/watch_as3|/html5player(?:-new)?|/base)?\.(?P<ext>[a-z]+)$',
+            r'.*?-(?P<id>[a-zA-Z0-9_-]+)(?:/[a-z]{2}_[A-Z]{2})?(?:/watch_as3|/html5player(?:-new)?|/base)?\.(?P<ext>[a-z]+)$',
             player_url)
         if not id_m:
             raise ExtractorError('Cannot identify player %r' % player_url)
